### DISCUSSION TITLE
GSA Mod Function Request - DinoColoring Brush Use - Restrictions With Mating

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#582] [Suggestions]
Please consider adding a restriction for utilizing the paint brush when the tame is in mating mode. This would prevent an exploit occurring where a stud male may be put on mating mode, and in the experimental painting phase, may breed an extravagant palette of various colors with multiple females without even needing to use up a single paint brush. 

In my screenshots, a tribemate and I coordinated our efforts with one stud male (originally brown) that bred three different females. While using the UI color picker to select colors for the male, the female was put on mating mode. After the first was bred, the colors were changed in the UI, and another female was put on mating mode. Rinse, repeat. Meanwhile, I could exit out of the UI after mating was completed and restart again on the same male, without ever having to consume the brush. The babies that were born all exhibited the colors of the male that was used as a stud. I have a video of the process that can be supplied on request. 
![2399830_20231124232734_1](https://github.com/gameserverapp/Platform/assets/44717221/30b9700b-55a5-46ee-9d53-285a48c64dce)
![2399830_20231124232728_1](https://github.com/gameserverapp/Platform/assets/44717221/aae83c4f-5345-4457-a0e7-3e8b0bcf0d5d)

**Comment by mvaessen** [comment]
In the upcoming mod there will be logic that will disable mating / wandering when the brush is used. 
So people should avoid painting mating dino's or it might stop the mating process.
